app/views/observations/form/_projects.html.erb

```diff
@@ -1,15 +1,45 @@
 <%# Projects section of create_observation form %>
+  <% if @suspect_checked_projects.any? %>
+    <% flash_error(:form_observations_there_is_a_problem_with_projects.t) %>
+
+    <div class="alert alert-danger"
+         id="project_messages">
+      <%= tag.p(
+        "#{:form_observations_projects_out_of_range.t(
+          date: @observation.when
+        )}:"
+      ) %>
+      <ul>
+        <% @suspect_checked_projects.each do |proj| %>
+        <%= tag.li("#{proj.title} (#{proj.start_date_str} -
+                    #{proj.end_date_str})") %>
+        <% end %>
+      </ul>
+      <%= tag.p(
+        :form_observations_projects_out_of_range_help.t(button: button_name)
```

Comment:
To avoid implying that we're creating another button, I'll probably change this to `(button_name: button_name)` (here and in `en.txt`)

Reply:
Done.